code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -230,6 +242,8 @@
 	var/reagents_per_slice = reagents.total_volume/slices_num //Figure out how much reagents each slice inherits (losing slices loses reagents)
 	for(var/i = 1 to (slices_num - slices_lost)) //Transfer those reagents
 		var/obj/slice = new slice_path(src.loc)
+		if(contents.len)
+			slice.contents += contents[1] //Our contents move to the slices
```

Comment:
I don't see where it's removing contents[1] from contents, it's just adding it to slice.contents.